PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/link/mysql/TestConnectionUtils.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.link.mysql;
+
+import com.intellij.icons.AllIcons;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTask;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.mysql.cj.jdbc.ConnectionImpl;
+
+import javax.swing.*;
+import java.sql.*;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class TestConnectionUtils {
+
+    public static boolean testConnection(String url, String username, String password) {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
```

Comment:
why we need to verify Class.forName("com.mysql.jdbc.Driver") here and there

Reply:
load driver when we want to test connection to mysql for the "test connection" button.